prusti-tests/tests/compiletest.rs

```diff
@@ -126,6 +126,7 @@ fn run_verification_base(group_name: &str, filter: &Option<String>) {
         TemporaryEnvVar::set("PRUSTI_FULL_COMPILATION", "true"),
         TemporaryEnvVar::set("PRUSTI_ENCODE_UNSIGNED_NUM_CONSTRAINT", "true"),
         TemporaryEnvVar::set("PRUSTI_QUIET", "true"),
+        TemporaryEnvVar::set("PRUSTI_ENABLE_GHOST_CONSTRAINTS", "true"),
```

Comment:
Let's only enable it per-test for now?

Reply:
In the meanwhile though we are not testing any part of Prusti with the feature disabled, and it being disabled will be the default for any user. We can check for regressions when we make the feature enabled by default, not now. (And of course you can see locally and in the already-executed CI run if it caused any issues yet.)